### PR TITLE
Add docs for new Segment app

### PR DESCRIPTION
**Pull request by lkostrowski**

### CODE REVIEW

File: docs/developer/app-store/apps/segment.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    id
+    productVariantId
+    productSku
+    variantName
+  }
+  number
+}
+```
+
+## Pre-requisites
```
Comment (peelar) [inline docs/developer/app-store/apps/segment.mdx:82]: **suggestion (blocking)**:

```suggestion
## Prerequisites
```

File: docs/developer/app-store/apps/segment.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+1. Create a [Segment account](https://app.segment.com)
+2. Create a new connection source and set it to `Node.js`.
+3. Name the source you want, e.g., "Saleor Orders."
```
Comment (peelar) [inline docs/developer/app-store/apps/segment.mdx:86]: **suggestion (non-blocking)**:

```suggestion
3. Name the source, e.g., "Saleor Orders."
```

File: docs/developer/app-store/apps/segment.mdx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    variantName
+  }
+  number
+}
+```
+
+## Pre-requisites
+
+1. Create a [Segment account](https://app.segment.com)
+2. Create a new connection source and set it to `Node.js`.
```
Comment (peelar) [inline docs/developer/app-store/apps/segment.mdx:85]: **suggestion (non-blocking)**:

If we can, I suggest linking to some Segment docs here.

File: docs/developer/app-store/apps/segment.mdx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+3. Name the source you want, e.g., "Saleor Orders."
+4. Copy visible _Write Key_ and paste it into the app configuration.
+
+## Configuration
+
+The app requires configuring only a single field - a Segment Write Key.
```
Comment (peelar) [inline docs/developer/app-store/apps/segment.mdx:91]: **suggestion (non-blocking)**:

```suggestion
The app requires configuring only one field - a [Segment Write Key](https://segment.com/docs/connections/find-writekey/).
```

File: docs/developer/app-store/apps/segment.mdx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+### Testing
+
+The easiest way to test if the app is configured is to perform following steps:
```
Comment (peelar) [inline docs/developer/app-store/apps/segment.mdx:95]: **suggestion (non-blocking)**:

```suggestion
The easiest way to test if the app is configured correctly is to perform the following steps:
```

**Review by peelar — CHANGES_REQUESTED** [review]

**Comment by lkostrowski** [comment]
@peelar applied fixes

**Review by 2can — APPROVED** [review]

**Review by peelar — APPROVED** [review]

**Pull request merged**